semgrep/tests/e2e/snapshots/test_semgrep_core_parse_error/test_rule_parser__failure__error_messages/invalid_python.py/error.txt

```diff
@@ -4,7 +4,6 @@ rules:
 - rules.assert-eqeq-is-ok
 - rules.eqeq-is-bad
 No .semgrepignore found. Using default .semgrepignore rules. See the docs for the list of default ignores: https://semgrep.dev/docs/cli-usage/#ignoring-files
-skipped 'targets/bad/invalid_python.py' [rule assert-eqeq-is-ok]: irrelevant_rule: target doesn't contain some elements required by rule 'assert-eqeq-is-ok'
```

Comment:
Do we still report files skipped due to irrelevant rule?

Reply:
Yes; this is because, having failed to parse it once, we now don't try to filter irrelevant rules